our-process/policies.md

```diff
@@ -26,6 +26,38 @@ needed.
 When submitting a package, please make sure that your GitHub notification
 settings are setup to notify you when you receive feedback on the review issue.
 
+## Submission volume and maintainer overlap
+
+To protect our volunteer peer review team and ensure quality reviews for all
+packages, we have policies regarding the volume of simultaneous submissions.
+
+### Unique point of contact requirement
+
+Each submission to pyOpenSci should have one unique point of contact per package.
```

Comment:
```suggestion
Each submission to pyOpenSci should have one point of contact per package.
```

super minor thing - using unique twice across these sentences is somewhat redundant and could be confusing: if there can only be one of something in a set, it is by definition unique within that set, so this makes me wonder if the use of unique here is supposed to mean something different than the use of unique in the next sentence.